app/scripts/controllers/ens/ens.js

```diff
@@ -0,0 +1,25 @@
+const EthJsEns = require('ethjs-ens')
+const ensNetworkMap = require('ethjs-ens/lib/network-map.json')
+
+class Ens {
+  constructor ({ network, provider } = {}) {
+    this._ethJsEns = new EthJsEns({
+      network,
+      provider,
+    })
+  }
+
+  lookup (ensName) {
+    return this._ethJsEns.lookup(ensName)
+  }
+
+  reverse (address) {
+    return this._ethJsEns.reverse(address)
+  }
+}
+
+Ens.getNetworkEnsSupport = function getNetworkEnsSupport (network) {
```

Comment:
Nit: This is a static function, right? Wouldn't this be the same as `static getNetworkEnsSupport (network) { ... }`

Not that it matters how it's written I guess, it just seemed a bit unconventional.

Reply:
I'll update this, good point